src/frontend/Komponenter/Journalføring/Standard/Behandlinger.tsx

```diff
@@ -0,0 +1,114 @@
+import React, { Dispatch, SetStateAction } from 'react';
+import { Button, HStack, Table, VStack } from '@navikt/ds-react';
+import { Fagsak } from '../../../App/typer/fagsak';
+import { Ressurs } from '../../../App/typer/ressurs';
+import DataViewer from '../../../Felles/DataViewer/DataViewer';
+import { formaterIsoDatoTid } from '../../../App/utils/formatter';
+import { behandlingStatusTilTekst } from '../../../App/typer/behandlingstatus';
+import { behandlingstypeTilTekst } from '../../../App/typer/behandlingstype';
+import LeggTilKnapp from '../../../Felles/Knapper/LeggTilKnapp';
+import { alleBehandlingerErFerdigstiltEllerSattPåVent } from '../../Personoversikt/utils';
+import { utledRiktigBehandlingstype } from '../Felles/utils';
+import { TrashIcon } from '@navikt/aksel-icons';
+import styled from 'styled-components';
+
+const StyledDataCell = styled(Table.DataCell)`
+    padding: 0;
+`;
+
+const FjernBehandlingButton = styled(Button)`
+    margin-right: 0.5rem;
+`;
+
+interface Props {
+    fagsak: Ressurs<Fagsak>;
+    settFeilmelding: Dispatch<SetStateAction<string>>;
+    skalOppretteNyBehandling: boolean;
+    settSkalOppretteNyBehandling: Dispatch<SetStateAction<boolean>>;
+}
+
+const Behandlinger: React.FC<Props> = ({
+    fagsak,
+    settFeilmelding,
+    skalOppretteNyBehandling,
+    settSkalOppretteNyBehandling,
+}) => {
+    const leggTilNyBehandlingForOpprettelse = (fagsak: Fagsak) => {
+        settFeilmelding('');
+        const kanOppretteNyBehandling = alleBehandlingerErFerdigstiltEllerSattPåVent(fagsak);
+
+        if (kanOppretteNyBehandling) {
+            settSkalOppretteNyBehandling(true);
+        } else {
+            settFeilmelding(
+                'Kan ikke opprette ny behandling. Denne fagsaken har en behandling som ikke er ferdigstilt.'
+            );
+        }
```

Comment:
Det er forskjellig oppførsel hvis det er klage vs vanlig her.
Ved klage kan man ha to behandlinger samtidig, men vi viser en advarsel om at det finnes åpen klage fra før

Reply:
Denne PRen er ikke ment for å ta seg av tilfeller med klage - tenke å legge til dette i en egen PR